package/gluon-web-model/javascript/gluon-web-model.js

```diff
@@ -219,6 +215,20 @@
 				parent.parentNode.style.display = (parent.options.length <= 1) ? 'none' : '';
 		}
 
+		var nodes = document.querySelectorAll('[data-exclusive-with]');
+		for (var i = 0, node; (node = nodes[i]) !== undefined; i++) {
+			var excusive_with = JSON.parse(node.getAttribute('data-exclusive-with'));
+
+			node.disabled = false;
+			for (var list_item of excusive_with) {
+				var el = document.getElementById(node.name + '.' + list_item);
+				node.disabled |= el.checked;
```

Comment:
```suggestion
				node.disabled ||= el.checked;
```

Also, this section of code could be implemented a bit more efficiently by having `node` disable `el` instead of vice-versa - this way, the inner loop could be skipped completely for unchecked `node`. We would need to iterate over `nodes` twice though (first to enable everything, then to disable what conflicts), so it would only make a difference for much longer exclusion lists (and might even make performance slightly worse in simple cases). So maybe it's not worth changing?